src/main/java/net/imagej/ops/transform/TransformNamespace.java

```diff
@@ -1207,4 +1207,59 @@ public String getName() {
 			ConcatenateViewWithAccessMode.class, source, concatenationAxis, mode);
 	}
 
+	@OpMethod(
+		op = net.imagej.ops.transform.realTransform.DefaultTransformView.class)
+	public <T extends RealType<T>> RandomAccessibleInterval<T> realTransform(
+		final RandomAccessibleInterval<T> in,
+		final InvertibleRealTransform transform)
+	{
+		final RandomAccessibleInterval<T> result =
+			(RandomAccessibleInterval<T>) ops().run(
+				net.imagej.ops.transform.realTransform.DefaultTransformView.class, in,
+				transform);
```

Comment:
This should be `ops().run(Ops.Transform.RealTransform.class, in, transform)`. That way, in the future, someone could write an op which is a specialized version of this and it could also be matched.

Please change this for all your other `realTransform(...)` namespace methods as well.